### ISSUE TITLE
Staff Form: Depended visibility dose not work on from Reset

### ISSUE THREAD

**Issue by ckleemann** [d120/pyophase#74]
If the reset button is used after to reset the form the depended Fields are still visible.
![bug](https://cloud.githubusercontent.com/assets/12739188/15966831/184e4afe-2f25-11e6-9e96-68f199a77fbe.png)

**Comment by jlauinger** [comment]
Could you rephrase the issue, please?

**Comment by exploide** [comment]
This means that for example the *Tutor for* field remains visible even when pressing *Reset* disables the *Tutor* checkbox (the one the *Tutor for* field depends on).
